templates/quickstart/TMQuickStartBYOL.template

```diff
@@ -0,0 +1,454 @@
+    ConstraintDescription: >-
+      Must begin with a letter and contain only alphanumeric characters. Min
+      length 1, max length 16
+  DatabaseAdminPassword:
+    NoEcho: true
+    Description: >-
+      The RDS Database administrator password. Must be 8-41 characters long and
+      can only contain alphanumeric characters or the following special
+      characters !^*-_+
+    Type: String
+    MinLength: 8
+    MaxLength: 41
+    AllowedPattern: '[a-zA-Z0-9!^*\-_+]*'
+    ConstraintDescription: >-
+      Can only contain alphanumeric characters or the following special
+      characters !^*-_+ Min length 8, max length 41
+  DeepSecuritySubnet:
+    Description: >-
+      Select an existing Subnet for Deep Security Manager. Must be a public
+      subnet contained the in VPC chosen above.
+    Type: 'AWS::EC2::Subnet::Id'
+    MinLength: '1'
+    MaxLength: '255'
+    AllowedPattern: '[-_a-zA-Z0-9]*'
+    ConstraintDescription: Subnet ID must exist in the chosen VPC
+  DeepSecurityAdminName:
+    Default: MasterAdmin
+    Description: The Deep Security Manager administrator username for Web Console Access.
+    Type: String
+    MinLength: 1
+    MaxLength: 16
+    AllowedPattern: '[a-zA-Z][a-zA-Z0-9]*'
+    ConstraintDescription: >-
+      Must begin with a letter and contain only alphanumeric characters. Min
+      length 1, max length 16
+  DeepSecurityAdminPass:
+    NoEcho: true
+    Description: >-
+      The Deep Security Manager administrator password. Must be 8-41 characters
+      long and can only contain alphanumeric characters or the following special
+      characters !^*-_+
+    Type: String
+    MinLength: 8
+    MaxLength: 41
+    AllowedPattern: '[a-zA-Z0-9!^*\-_+]*'
+    ConstraintDescription: >-
+      Can only contain alphanumeric characters or the following special
+      characters !^*-_+ Min length 8, max length 41
+  LicenseKey:
+    Description: >-
+      If you have a Deep Security Enterprise license key, enter it here.
+      Including dashes (e.g. AP-E9RM-99WHE-B5UR5-BV8YB-HVYM8-HYYVG). If you did
+      not purchase all modules please leave blank and enter your key in the
+      console after launch.
+    Type: String
+    MinLength: 0
+    MaxLength: 37
+    AllowedPattern: >-
+      (?:[A-Z0-9]{2}-[A-Z0-9]{4}-[A-Z0-9]{5}-[A-Z0-9]{5}-[A-Z0-9]{5}-[A-Z0-9]{5}-[A-Z0-9]{5})?
+    ConstraintDescription: Key can only contain ASCII characters.
+  QSS3BucketName:
+    AllowedPattern: ^[0-9a-zA-Z]+([0-9a-zA-Z-.]*[0-9a-zA-Z])*$
+    ConstraintDescription: Quick Start bucket name can include numbers, lowercase
+      letters, uppercase letters, periods (.), and hyphens (-). It cannot start or
+      end with a hyphen (-).
+    Default: trend-micro-quick-start
+    Description: S3 bucket name for the Quick Start assets. Quick Start bucket name
+      can include numbers, lowercase letters, uppercase letters, and hyphens (-).
+      It cannot start or end with a hyphen (-).
+    Type: String
+  QSS3KeyPrefix:
```

Comment:
This whole parameter should look like this (AllowedPattern, ConstraintDescription, Description), but with a Default value of "trendmicro/deepsecurity/latest/": https://github.com/aws-quickstart/quickstart-linux-bastion/blob/develop/templates/linux-bastion-master.template#L225